GIT_WORKFLOW_AND_HISTORY_STYLE_GUIDE.md

```
@@ -0,0 +1,500 @@
+If it helps navigation, you may prefix the subject with a short area name:
+
+* `CI: Fix flaky test on Linux`
+* `Doc: Remove broken link`
+* `MainWindow: Change title`
+* `main.c: Remove unused helper`
+* `Context: Stop using deprecated field`
+
+Rules for area prefixes:
+
+* Keep them short and stable.
+* Use Title Case (or a file name), followed by `: `.
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)